packages/apps/examples/nft-sdk/src/router.ts

```diff
@@ -34,4 +34,9 @@ router.get(
   nftController.getContractEventLogs
 );
 
+router.get(
+  '/api/getNFTsByWalletAndContracts/:address/:contractOne',
```

Comment:
I was fighting with typescript a lot with that, probably missing something obvious, but I kept it a path param to keep it simple for now since this is just for example purposes

Reply:
ok, was able to figure it out! updated